changelogs/fragments/535-ntpregression-test-fix.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Comment:
Please change the classification to "trivial". This is because, from the users' perspective, our regression tests are not related to the functionality of the resource module collection. (Any changes in the "test" directory are usually classified this way.)